src/index.html

```diff
@@ -3,11 +3,238 @@
+              </p>
+              <h3 class="compare__bike-price">$ 2 090</h3>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="main__details details">
+        <div class="details__content-wrapper">
+          <h2 class="details__heading">The Details</h2>
+
+          <div class="details__photo-row">
+            <img
+              src="images/photos/details-1.png"
+              alt="handlebars with phone attached"
+              class="details__photo details__photo--cols--1-4"
+            >
+            <img
+              src="images/photos/details-2.png"
+              alt="handlebars with phone attached from the back"
+              class="details__photo details__photo--cols--5-6"
+            >
+          </div>
+          <h3 class="details__title">Auto Unlock</h3>
+          <p class="details__description">
+            The app senses when you're nearby to unlock automatically.
+            GPS tracking so you know where your bike is
+            and can track it anytime.
+          </p>
+
+          <div class="details__photo-row">
+            <img
+              src="images/photos/details-3.png"
+              alt="bike with red rear light on"
+              class="details__photo details__photo--cols--1-4"
+            >
+            <img
+              src="images/photos/details-4.png"
+              alt="bike with white headlight light on"
+              class="details__photo details__photo--cols--5-6"
+            >
+          </div>
+          <h3 class="details__title">Range & Integrated lights</h3>
+          <p class="details__description">
+            The removable battery has up to 70km battery autonomy and
+            weighs only 2.4 kg. Lights integrated into the frame
+            give you always-on visibility day and night.
+          </p>
+
+          <div class="details__photo-row">
+            <img
+              src="images/photos/details-5.png"
+              alt="bike handbrake"
+              class="details__photo details__photo--cols--1-2"
+            >
+            <img
+              src="images/photos/details-6.png"
+              alt="lower half of the bike exposing disc brakes"
+              class="details__photo details__photo--cols--3-6"
+            >
+          </div>
+          <h3 class="details__title">Hydraulic disc brakes & Lightweight</h3>
+          <p class="details__description">
+            Brakes with total stopping power the second you make contact.
+            The removable battery has up to 70km battery autonomy and
+            weighs only 2.4 kg. Lights integrated into the frame
+            give you always-on visibility day and night.
+          </p>
+
+          <a href="#" class="details__explore-button">Explore</a>
+        </div>
+      </section>
+
+      <section class="main__contact contact">
+        <h2 class="contact__heading">Contact us</h2>
+
+        <div class="contact__wrapper">
+          <form
```

Comment:
> 14. Form shouldn't be submitted if some of the fields are not filled
> 15. Page shouldn't be reloaded on form submit (https://developer.mozilla.org/en-US/docs/Web/API/Event/preventDefault)

Reply:
@LukaszGryziewicz your form does not perform validation
please implement
you can prevent the page reload with .js:


```
const form = document.querySelector('<form selector>');

form.addEventListener('submit', function(event) {
  event.preventDefault();
  form.reset();
});
```